GetStream/stream-video-flutter
# [283] fix user names and images

### 🎯 Goal

Closes #283 

### 🧪 Testing

- Launch `example` app on Web
- Select any user (for instance `Willard Hessel`)
- Start a call
- Join to that call from another client (iOS or Android) using a different user.
- Make sure names and avatars are presented



### ☑️Contributor Checklist

#### General

### ☑️Reviewer Checklist
